docs/index.md

```diff
@@ -2,120 +2,82 @@
 layout: default
 ---
 
-Text can be **bold**, _italic_, or ~~strikethrough~~.
+**SciSpaCy is a Python package containing [SpaCy](https://spacy.io/) models for processing _biomedical_ or _clinical_ text.**
 
-There should be whitespace between paragraphs.
-
-There should be whitespace between paragraphs. We recommend including a README, or a file with information about your project.
-
-# Header 1
-
-This is a normal paragraph following a header. GitHub is a code hosting platform for version control and collaboration. It lets you and others work together on projects from anywhere.
-
-## Header 2
-
-> This is a blockquote following a header.
->
-> When something is important enough, you do it even if the odds are not in your favor.
-
-### Header 3
-
-```js
-// Javascript code with syntax highlighting.
-var fun = function lang(l) {
-  dateformat.i18n = require('./lang/' + l)
-  return true;
-}
+## Installing
+```python
+pip install <Mark add package here>
+pip install <add paths to models here.>
 ```
+## Models
+
+### Performance
+
+Our models achieve performance within 1% of published state of the art dependency parsers and within 0.2% accuracy of state of the art biomedical POS taggers.
+
+| model          | UAS | LAS   | POS   | NER | Web UAS
+|:---------------|:----|:------|:------|:---|:---|
+| en_core_sci_sm | good| nice  |       |    |    |
+| en_core_sci_md | good| nice  |       |    |    |
+
+
+### Example Usage
+
+```python
+import scispacy
+import spacy
+
+nlp = spacy.load("en_core_sci_sm")
+text = """
+Myeloid derived suppressor cells (MDSC) are immature 
+myeloid cells with immunosuppressive activity. 
+They accumulate in tumor-bearing mice and humans 
+with different types of cancer, including hepatocellular 
+carcinoma (HCC).
+"""
+doc = nlp(text)
```

Comment:
you also want to call scispacy.custom_tokenizer.remove_new_lines on the raw text here before passing it on to nlp. This is not strictly necessary, but is expected to improve the results for pdf parsed text. The reason it isn't part of the actual model is that spacy has a strict contract of being lossless, and this function removes some extraneous spaces/hyphens

Reply:
Gotcha - i'll leave that out as it's fairly specific to our use case